src/box/vy_scheduler.c

```diff
@@ -1116,6 +1117,140 @@ vy_task_write_run(struct vy_task *task, bool no_compression)
 	return -1;
 }
 
+/**
+ * Table of symbols:
+ * R - Worst-case zero-result point lookup cost (in runs);
+ * W_u - number of unfiltered runs;
+ * W_f - number of filtered runs;
+ * t_r - Adopted run_size_ratio_coeff.
+ *
+ * Note:
+ * We should consider the fact that
+ * current range doesn't yet contains newly created run.
+ *
+ * @param lsm
+ * @param range
+ * @param new_run_stmts_count statements count in the newly created after
+ * dump/compaction run
+ * @param W total number of runs in the range
```

Comment:
Let's name the parameter properly and create an alias for it locally.

Reply:
Done.